src/Datadog.Trace/Configuration/EnvironmentConfigurationSource.cs

```diff
@@ -13,5 +13,5 @@ public override string GetString(string key)
         {
             return Environment.GetEnvironmentVariable(key);
         }
-    }
```

Comment:
nit: Looks like the whitespace got thrown off, can you fix this?

Reply:
Absolutely! :)